### DISCUSSION TITLE
How I can put custom type in tada

### DISCUSSION THREAD

**Discussion by salemkode** [0no-co/gql.tada#166] [Q&A] [answered]
I'm currently working with a server that utilizes the bytea type in its GraphQL schema. However, when I attempt to fetch data using queries and utilize the "tada" library to generate TypeScript runtime types, I encounter an issue where the bytea type is not properly handled, resulting in a never type being returned.

In order to address this issue, I'm seeking a solution to create a custom alias for the bytea type in TypeScript. Can you provide guidance on how to create such a custom alias and integrate it effectively with the types generated by "tada"?

![image](https://github.com/0no-co/gql.tada/assets/58360393/4ba74c5a-603a-45ef-a5a4-d83560bc17e9)

**Comment by kitten** [comment, marked as answer]
Are you maybe looking for this docs section? https://gql-tada.0no.co/get-started/installation#customizing-scalar-types

**Reply by salemkode** [reply]
Thank you so much for sharing the link
